pkg/terminal/command.go

```diff
@@ -2676,7 +2677,15 @@ func printfile(t *Term, filename string, line int, showArrow bool) error {
 		arrowLine = line
 	}
 
-	file, err := os.Open(t.substitutePath(filename))
+	var file *os.File
+	path := t.substitutePath(filename)
+	if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
+		path, err = debuginfod.GetSource(t.client.BuildID(), filename)
```

Comment:
Only overwrite path if there is no error, if there is an error let the code continue as normal. Should this use a cache or is debuginfod-find supposed to be fast?

Reply:
We overwrite path if the path we get isn't available. `debuginfod-find` is fast enough and has it's own cache.

Reply:
What I mean is, something like this:

```
path2, err := debuginfod.GetSource...
if err == nil {
    path = path2
}
```

so that the error message doesn't talk about debuginfod just because the source file can't be found and debuginfod-find can't be used.

Reply:
Ah yes. We can do something like that so the errors are consistent.